docs/how-to-guides/upgrade-openebs.md

```diff
@@ -0,0 +1,248 @@
+# Upgrading and migrating OpenEBS data plane components
+
+## Contents
+
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)
+* [Step 1: Upgrade OpenEBS control plane components](#step-1-upgrade-openebs-control-plane-components)
+* [Step 2: Upgrade OpenEBS data plane components](#step-2-upgrade-openebs-data-plane-components)
+	* [Upgrade cStor pools](#upgrade-cstor-pools)
+	* [Upgrade cStor volumes](#upgrade-cstor-volumes)
+* [Step 3: Verify](#step-3-verify)
+* [Summary](#summary)
+* [Additional resources](#additional-resources)
+
+## Introduction
+
+This guide provides steps to upgrade OpenEBS control plane and data plane
+components from one version to another.
+
+In this document we are going to update from `v1.12.0` to `v2.2.0`.
+
+## Prerequisites
+
+To upgrade the OpenEBS data plane components, we need the following:
+
+* A Kubernetes cluster accessible via `kubectl`.
+
+* [OpenEBS](../configuration-reference/components/openebs.md) installed. You can
+  check if OpenEBS is indeed in expected version:
+    ```bash
+    $ kubectl get pods -n openebs -l openebs.io/version=1.12.0
+    ```
+
+* Highly recommended to schedule a downtime for the applications consuming the
+  OpenEBS PV and make sure to take a backup of the data before starting the
+  below upgrade procedure. Lokomotive provides
+  [Velero](../configuration-reference/components/velero.md) component for backup
+  and restore.
+
+* Ensure the cluster and OpenEBS volumes are is in healthy state before proceeding.
+    ```bash
+    $ lokoctl health
+    Node                    Ready    Reason          Message
+
+    alpha-controller-0      True     KubeletReady    kubelet is posting ready status
+    alpha-large-worker-0    True     KubeletReady    kubelet is posting ready status
+    Name      Status    Message              Error
+
+    etcd-0    True      {"health":"true"}
+
+
+   $ kubectl get cstorpools -n openebs # Status should be Healthy.
+
+   NAME                                ALLOCATED   FREE   CAPACITY   STATUS    READONLY   TYPE      AGE
+   cstor-pool-openebs-replica-1-w3r7   6.98G       437G   444G       Healthy   false      striped   4d22h
+
+
+   $ kubectl get cstorvolume -n openebs # Status should be Healthy.
+
+   NAME                                       STATUS    AGE     CAPACITY
+   pvc-183dc3df-a7d9-4273-a8e2-7f66d2e19f4e   Healthy   3d20h   50Gi
+   pvc-5d4b4c2b-2ed5-4e75-aeb9-fbd2918ebb78   Healthy   3d20h   50Gi
+   ```
+## Steps
+
+Set the following environment variables in the terminal to assist in our upgrade:
+```bash
+export OPENEBS_NEW_VERSION=2.2.0
+export OPENEBS_OLD_VERSION=1.12.0
```

Comment:
First old, then new. Otherwise it's counter intuitive and may indicate _downgrade_ process.
```suggestion
export OPENEBS_OLD_VERSION=1.12.0
export OPENEBS_NEW_VERSION=2.2.0
```